src/Sentry.Extensions.AI/SentryAIUtil.cs

```diff
@@ -0,0 +1,22 @@
+using Sentry.Internal;
+
+namespace Sentry.Extensions.AI;
+
+internal static class SentryAIUtil
```

Comment:
Potentially this class could be renamed something like `FICCSpanTracker` and could encapsulate all the logic related to storing/retrieving the FICC span... that way none of the code outside this class would need to know implementation details like the `SentryAIConstants.SentryActivitySpanAttributeName` dictionary key (that could be kept private to this class) or that the Fused extensions are being used under the hood.

Not sure if you like that idea or not... maybe worth playing with to see if it results in tidier code.